assets/js/modules/analytics-4/datastore/report.js

```diff
@@ -227,6 +232,17 @@ const baseSelectors = {
 			return undefined;
 		}
 
+		const hasReportError = select(
```

Comment:
Note, I had started implementing this in #6212 but realised it could be brought into this PR in order to get the code merged to `develop` sooner.